src/request/providers/node.ts

```diff
@@ -473,7 +468,7 @@ export default function node(url: string, options: NodeRequestOptions = {}): Tas
 							if (contentEncodings.length) {
 								const encoding = contentEncodings.pop()!.trim();
 
-								if (encoding === '' || encoding === 'identity') {
+								if (encoding === '' || encoding.toLowerCase() === 'none' || encoding === 'identity') {
```

Comment:
I guess that would be alright wouldn't it. I was afraid of lowercasing them all and changing the current behavior but since everything being checked for now is lowercased anyway, I think its ok.